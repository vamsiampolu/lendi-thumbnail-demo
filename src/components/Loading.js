import React from 'react'
import { Dimmer, Loader } from 'semantic-ui-react'


export default function Loading(props) {
  return (<Dimmer page active>
    <Loader size='huge'> Loading </Loader>
  </Dimmer>)
}
