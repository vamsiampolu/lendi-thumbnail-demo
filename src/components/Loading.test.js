import React from 'react'
import renderer from 'react-test-renderer'
import Loading from './Loading'

describe('Loading', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Loading />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
 })
