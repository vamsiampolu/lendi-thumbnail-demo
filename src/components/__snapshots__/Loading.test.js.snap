exports[`Loading renders correctly 1`] = `null`;
